private/dump_new_fish_data.py

```diff
@@ -703,6 +714,12 @@ def get_location_key(spot):
             if folklore is not None:
                 folklore_key = folklore.key
 
+        min_rating = 0
+        if fish.scrip is not None:
+            min_rating = get_min_rating(fish)
+        elif fish.reducible:
+            min_rating = 1
```

Comment:
I would prefer the data be recorded first in the YALM generator section above.  This function effectively copies all that data into the JSON representation (which turns into javascript data).

Basically, the YALM output becomes a starting point for managing the fish data whenever I finally cut over to this builder.

Reply:
So upon looking further, if I'm reading correctly, it looks like you're already returning the whole row to the YAML from the scan tasks, so it's probably easier to just grab the thresholds from said rows rather than having a separate calc function. get_min_rating was removed, and per the comment on having a separate value:
`'scrip'`/`'satisfaction'`/`'reduce'` now have values of `LowCollectability`/`Collectability{High}`/`1` respectively if the fish fits the category, and None otherwise